tools/downloader/pytorch_to_onnx.py

```diff
@@ -22,6 +22,12 @@ def positive_int_arg(values):
     return result
 
 
+def model_parameters(parameters):
+    if not parameters:
+        return dict()
+    return dict((param, eval(value)) for param, value in (element.split('=') for element in parameters.split(',')))
```

Comment:
Thanks for comment about `eval` safeness!
But first one could not be applied.

Reply:
> But first one could not be applied.

What do you mean?

Reply:
If you mean `eval(parameters)` it could not be done without major changes in `--model-params` structure.

Reply:
I know. So what?

Reply:
Because chosen structure is more natural for reading and adding in config and I dont want to change it just for simplifying few lines of code. 
If you have better solution please show me and I would implement it if it fits well.

Reply:
The primary reason is not to simplify code, but to make the syntax more universal. With your syntax, a parameter value can't contain commas, whereas a Python expression certainly can.

> If you have better solution please show me and I would implement it if it fits well.

Alright, how about this then:

1. Change the name to `--model-param`.
2. Add `action='append'`.
3. Parse in basically the same way, except without the comma splitting.

This makes the syntax universal without changing it much.

Reply:
Done